@navikt/core/css/darkside/pagination.darkside.css

```diff
@@ -7,65 +7,28 @@
   padding: 0;
   list-style: none;
   display: flex;
-  gap: var(--a-spacing-3);
+  gap: var(--ax-spacing-3);
 }
 
 .navds-pagination__ellipsis {
   display: flex;
   align-items: center;
   justify-content: center;
-  width: var(--a-spacing-12);
+  width: 3rem;
   cursor: default;
 }
 
 .navds-pagination--small .navds-pagination__ellipsis {
-  width: var(--a-spacing-8);
+  width: 2rem;
 }
 
 .navds-pagination--xsmall .navds-pagination__ellipsis {
-  width: var(--a-spacing-6);
+  width: 1.5rem;
 }
 
-.navds-pagination__item {
-  color: var(--ac-pagination-text, var(--a-text-default));
-}
-
-.navds-pagination--medium .navds-pagination__item {
-  border-radius: var(--a-border-radius-medium);
-}
-
-.navds-pagination__item[aria-current="true"] {
-  background-color: var(--ac-pagination-selected-bg, var(--a-surface-action-selected));
-  color: var(--ac-pagination-selected-text, var(--a-text-on-action));
-}
-
-.navds-pagination__item[aria-current="true"]:focus-visible {
-  box-shadow:
-    inset 0 0 0 1px var(--a-surface-default),
-    var(--a-shadow-focus);
-}
-
-@supports not selector(:focus-visible) {
-  .navds-pagination__item[aria-current="true"]:focus {
-    box-shadow:
-      inset 0 0 0 1px var(--a-surface-default),
-      var(--a-shadow-focus);
-  }
-}
-
-.navds-pagination__prev-next {
-  display: flex;
-  align-items: center;
```

Comment:
There's some height issue with small and xsmall:
![image](https://github.com/user-attachments/assets/f8a91259-c93b-4dcb-a8d2-55fbbaae41e7)
Seems to be a line-height issue. It doesn't happen with old css because of this:
```css
.navds-pagination__prev-next {
  display: flex;
```
But there are probably several ways to fix it.

Reply:
Can't seem to replicate this issue, what browser/story is this?

Reply:
Chrome. It's most noticeable on xsmall. Doesn't matter the story, for example "X Small".

Reply:
I can replicate in FF and Edge as well.

Reply:
Seems to be caused by flex-inline in button + Chevron-icon. The "baseline" gets skewed with icon, if i remove it its ok. Should be easy to fix 👍

Reply:
Adding this solves it:
```
& li {
    list-style: none;
    display: flex;
  }
```

`list-style: none;` is not strictly needed, but think its good practice just in case a browser suddenly adds some `::marks`